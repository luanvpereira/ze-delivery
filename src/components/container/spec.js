import React from 'react';
import { mount } from 'enzyme';

import Container from './';

describe('<Container/>', () => {
	it('should render a <Container/>', () => {
		const component = mount(
			<Container className="CustomClass">
				<p>Corpo</p>
			</Container>
		);

		expect(component).toMatchSnapshot();
	});
});
