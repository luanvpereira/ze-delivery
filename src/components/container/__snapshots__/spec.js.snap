// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container/> should render a <Container/> 1`] = `
<Container
  className="CustomClass"
>
  <div
    className="container CustomClass"
  >
    <p>
      Corpo
    </p>
     
  </div>
</Container>
`;
